docs/modules/ROOT/pages/api/account.adoc

```diff
@@ -142,14 +145,17 @@ Initializes the account with the given public key, and registers the ISRC6 inter
 
 Emits an {OwnerAdded} event.
 
+[#Account-Utilities]
+==== Utilities
+
```

Comment:
not sure if this is introduced in this PR or it comes from another, but i think we agreed on not having an Utilities section (not sure where, but maybe in the access PR)